image/templates/helm/stackrox-secured-cluster/templates/collector.yaml.htpl

```diff
@@ -234,6 +248,13 @@ spec:
       - hostPath:
           path: /
         name: host-root-ro
+    {{ if eq ._rox.env.openshift 4 }}
+      - hostPath:
+        path: /usr/share
```

Comment:
Needs to indented?

Reply:
I don't know about that. 
It looks like we align the whitespace trimming `-` with the yaml entries `-`, so IIUC, this should be the right indent, right? 
That said, is there something like a helm linter we could utilize to enforce our defined style?

Reply:
I wasn't referring to style, but not semantics. `path` just be a field within the `hostPath` dict? As in two lines above your change.

Reply:
Interestingly, the helm chart worked as intended, but I have still corrected the `path` entry. Sharp eye! :smile: